plugins/source/slack/Makefile

```diff
@@ -0,0 +1,40 @@
+# Test unit
+.PHONY: test
+test:
+	go test -timeout 3m ./...
+
+# Install tools
+.PHONY: install-tools
+install-tools:
```

Comment:
Updated, though since this was copy-pasted from another plugin, this will be an issue elsewhere as well. I'm seeing fixing this for other plugins as out-of-scope for this PR.